refinery/core/serializers.py

```diff
@@ -24,17 +24,23 @@ class DataSetSerializer(serializers.ModelSerializer):
     is_clean = serializers.SerializerMethodField()
 
     def get_is_owner(self, data_set):
-        owner = data_set.get_owner()
         try:
-            user_request = self.context.get('request').user
-        except AttributeError as e:
-            logger.error("Request is missing a user: %s", e)
-            return False
-        return user_request == owner
+            data_set.is_owner
```

Comment:
for the data set view api, it's faster to pass along a value because we are prefetching perms.